text/1001-deprecate-named-inject.md

```diff
@@ -0,0 +1,60 @@
+---
+stage: accepted
+start-date: 2023-12-26T00:00:00.000Z
+release-date:
+release-versions:
+teams: # delete teams that aren't relevant
+  - framework
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/1001
+project-link:
+---
+
+<!---
+Directions for above:
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+-->
+
+# Deprecate named inject 
+
+## Summary
+
+As of [`ember-source@4.1`](https://blog.emberjs.com/ember-4-1-released) (and [RFC#752](https://github.com/emberjs/rfcs/pull/752)),  `inject` is an old alias that's no longer needed
+
+## Motivation
+
+`import { service } from '@ember/service'`
+makes more sense than 
+`import { inject as service } from '@ember/service'`
+
+This allows us to slim down our public API surface area to more of _what's needed_.
+
+
+## Transition Path
+
+Most folks can do a mass find and replace switch from `inject as service` to just `service`.
```

Comment:
There are more details needed here -- `import { inject as service } from '@ember/service'` was a convention, some people also did `import { inject } from '@ember/service'` and will need more work to cleanup. 

There's also `import { inject as service }, Service from '@ember/service'` and `import Service, { inject } from '@ember/service'. This may warrant a codemod.

Reply:
all of those are still find and replaceable.

In any case, you're right, it's a super easy codemod as well -- probably < 100 lines

Reply:
Here is one I just whipped up with < 50 lines of new code:

https://astexplorer.net/#/gist/119f88339ea024e7cde63c71f52ce216/4d128a1239cbb56e00a69d3f710d67c20ed0e431

Reply:
codemod added!